File: src/main/java/com/aymanosman/racket/plugin/psi/impl/RacketNamedElementImpl.java
package com.aymanosman.racket.plugin.psi.impl;

import com.aymanosman.racket.plugin.psi.RacketNamedElement;
import com.intellij.extapi.psi.ASTWrapperPsiElement;
import com.intellij.lang.ASTNode;
import org.jetbrains.annotations.NotNull;

public abstract class RacketNamedElementImpl extends ASTWrapperPsiElement implements RacketNamedElement {
    public RacketNamedElementImpl(@NotNull ASTNode node) {
        super(node);
    }
}

File: src/main/kotlin/org/racket/lang/core/psi/impl/RacketNamedElementImpl.kt
package org.racket.lang.core.psi.impl;

import com.intellij.extapi.psi.ASTWrapperPsiElement;
import com.intellij.lang.ASTNode;
import org.jetbrains.annotations.NotNull;
import org.racket.lang.core.psi.RacketNamedElement;

public abstract class RacketNamedElementImpl extends ASTWrapperPsiElement implements RacketNamedElement {
    public RacketNamedElementImpl(@NotNull ASTNode node) {
        super(node);
    }
}
